src/lua/functions/creatures/player/player_functions.cpp

```diff
@@ -1767,6 +1767,7 @@ int PlayerFunctions::luaPlayerGetStorageValueByName(lua_State* L) {
 		return 0;
 	}
 
+	g_logger().warn("The function 'player:getStorageValueByName' is deprecated and will be removed in future versions, please use KV system");
```

Comment:
isn't better to change to debug?

Reply:
No, it must be reported in warning mode, as it is a warning that it will be removed in a future release. The peoples need to stop using.